2016-12-05 12:48:43.060 +01:00 [Error] Ett fel hanterades i GlobalExceptionFilter: '"An error occurred while executing the command definition. See the inner exception for details."'. Stacktrace: "   vid System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   vid System.Data.Entity.Core.Objects.Internal.ObjectQueryExecutionPlan.Execute[TResultType](ObjectContext context, ObjectParameterCollection parameterValues)
   vid System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   vid System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   vid System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   vid System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   vid System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   vid System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   vid System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   vid System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   vid ODL.DataAccess.Repositories.ResultatenhetRepository.GetAll() i C:\repo\ODLService\ODL.Service\ODL.DataAccess\Repositories\ResultatenhetRepository.cs:rad 39
   vid ODL.ApplicationServices.OrganisationService.GetResultatenheter() i C:\repo\ODLService\ODL.Service\ODL.ApplicationServices\OrganisationService.cs:rad 39
   vid ODL.Service.Controllers.OrganisationController.Get() i C:\repo\ODLService\ODL.Service\src\ODL.Service\Controllers\OrganisationController.cs:rad 23
   vid lambda_method(Closure , Object , Object[] )
   vid Microsoft.AspNetCore.Mvc.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   vid Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionFilterAsync>d__28.MoveNext()"
2016-12-05 12:49:28.857 +01:00 [Error] Ett fel hanterades i GlobalExceptionFilter: '"An error occurred while executing the command definition. See the inner exception for details."'. Stacktrace: "   vid System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   vid System.Data.Entity.Core.Objects.Internal.ObjectQueryExecutionPlan.Execute[TResultType](ObjectContext context, ObjectParameterCollection parameterValues)
   vid System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   vid System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   vid System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   vid System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   vid System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   vid System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   vid System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   vid System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   vid ODL.DataAccess.Repositories.ResultatenhetRepository.GetAll() i C:\repo\ODLService\ODL.Service\ODL.DataAccess\Repositories\ResultatenhetRepository.cs:rad 39
   vid ODL.ApplicationServices.OrganisationService.GetResultatenheter() i C:\repo\ODLService\ODL.Service\ODL.ApplicationServices\OrganisationService.cs:rad 39
   vid ODL.Service.Controllers.OrganisationController.Get() i C:\repo\ODLService\ODL.Service\src\ODL.Service\Controllers\OrganisationController.cs:rad 23
   vid lambda_method(Closure , Object , Object[] )
   vid Microsoft.AspNetCore.Mvc.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   vid Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionFilterAsync>d__28.MoveNext()"
2016-12-05 13:07:47.710 +01:00 [Error] Ett fel hanterades i GlobalExceptionFilter: '"Sekvensen inneh√•ller inga element"'. Stacktrace: "   vid System.Linq.Enumerable.Single[TSource](IEnumerable`1 source)
   vid ODL.DataAccess.Repositories.PersonRepository.GetByPersonnummer(String personnummer) i C:\repo\ODLService\ODL.Service\ODL.DataAccess\Repositories\PersonRepository.cs:rad 38
   vid ODL.ApplicationServices.OrganisationService.GetResultatenhetByPersonnummer(String personnummer) i C:\repo\ODLService\ODL.Service\ODL.ApplicationServices\OrganisationService.cs:rad 23
   vid ODL.Service.Controllers.OrganisationController.GetResultatenhet(String personnummer) i C:\repo\ODLService\ODL.Service\src\ODL.Service\Controllers\OrganisationController.cs:rad 30
   vid lambda_method(Closure , Object , Object[] )
   vid Microsoft.AspNetCore.Mvc.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   vid Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionFilterAsync>d__28.MoveNext()"
